tests/common/widgets/infinite-hits/index.ts

```diff
@@ -22,5 +23,6 @@ export function createInfiniteHitsWidgetTests(
   describe('InfiniteHits widget common tests', () => {
     createOptimisticUiTests(setup, { act, skippedTests });
     createInsightsTests(setup, { act, skippedTests });
```

Comment:
the insights tests are failing because more infinite hits are rendered now

Reply:
Yup that's fixed.

Reply:
Yup that's fixed.